### PR TITLE
[ES] Translation correction `Beatmap/Beatmap_description`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Beatmap_description/es.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ---
-no_native_review: true
+no_native_review: false
```
Comment (The-Last-Cookie) [inline wiki/Beatmap/Beatmap_description/es.md:2]: This change is not necessary. The tag will be removed on merge by a maintainer (if someone other than the author reviewed the article, which is the case here as it seems to me). I would recommend you to mention the removal of the tag in the PR's description for better visibility.

Comment (UnknownOsu) [reply wiki/Beatmap/Beatmap_description/es.md:2]: Oh, I didn't know about that. I did that on all the articles I submitted and they had that tag. I will keep that in mind for future articles I submit.
Sorry about that.

Comment (The-Last-Cookie) [reply wiki/Beatmap/Beatmap_description/es.md:2]: No worries! I wanted to help you get used to the wiki procedures more easily.

File: wiki/Beatmap/Beatmap_description/es.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 ---
 
-# Descripción del beatmap
+# Descripción del mapa
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:5]:
```suggestion
# Descripción del beatmap
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 
-La **descripción del beatmap** es un área pequeña debajo de la página de un [beatmap](/wiki/Beatmap) que brinda a los jugadores información sobre el beatmap usando [BBCode](/wiki/BBCode). Por lo general, destaca algunos datos sobre el beatmapque su creador consideró lo suficientemente importante, o se usa para publicitar otros proyectos de mapping.
+La **descripción del mapa** es un área pequeña debajo de la página de un [mapa](/wiki/Beatmap) que brinda a los jugadores información sobre el mapa usando [BBCode](/wiki/BBCode). Por lo general, destaca algunos datos sobre el mapa que su creador consideró lo suficientemente importante, o se usa para publicitar otros proyectos de mapping.
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:7]:
```suggestion
La **descripción del beatmap** es un área pequeña debajo de la página de un [beatmap](/wiki/Beatmap) que brinda a los jugadores información sobre el beatmap usando [BBCode](/wiki/BBCode). Por lo general, destaca algunos datos sobre el beatmap que su creador consideró lo suficientemente importante, o se usa para publicitar otros proyectos de mapping.
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
 
 ## Usos e información
 
-Las descripciones populares de beatmaps [clasificados](/wiki/Beatmap/Category#clasificados) a menudo incluyen lo siguiente:
+Las descripciones populares de mapas [clasificados](/wiki/Beatmap/Category#clasificados), a menudo, incluyen lo siguiente:
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:11]:
```suggestion
Las descripciones populares de beatmaps [clasificados](/wiki/Beatmap/Category#clasificados), a menudo, incluyen lo siguiente:
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 5]
```diff
@@ -1,25 +1,25 @@
...
 
 - Una lista de todas las [dificultades de invitados](/wiki/Beatmap/Guest_difficulty) y sus creadores
-- Información que ayudaría a un jugador a decidir si quiere descargar el beatmap o no
+- Información que ayudaría a un jugador a decidir si quiere descargar el mapa o no
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:14]:
```suggestion
- Información que ayudaría a un jugador a decidir si quiere descargar el beatmap o no
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
 - Una lista de ediciones o registro de cambios
 
 ### Imágenes
 
-Muchas descripciones de beatmaps contienen al menos una imagen (a menudo un [GIF](https://es.wikipedia.org/wiki/Graphics_Interchange_Format) animado de algún tipo) que se usa para expresar e ilustrar al creador, la canción y/o beatmap de una manera llamativa. También pueden servir como título o encabezamiento para agregar estilo y calidad. Para saber cómo agregar una imagen, consulta la [sección de imágenes del artículo de BBCode](/wiki/BBCode#imágenes).
+Muchas descripciones de mapas contienen al menos una imagen (a menudo, un [GIF](https://es.wikipedia.org/wiki/Graphics_Interchange_Format) animado de algún tipo) que se usa para expresar e ilustrar al creador, la canción y/o mapa de una manera llamativa. También pueden servir como título o encabezamiento para agregar estilo y calidad. Para saber cómo agregar una imagen, consulta la [sección de imágenes del artículo de BBCode](/wiki/BBCode#imágenes).
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:19]:
```suggestion
Muchas descripciones de beatmaps contienen al menos una imagen (a menudo un [GIF](https://es.wikipedia.org/wiki/Graphics_Interchange_Format) animado de algún tipo) que se usa para expresar e ilustrar al creador, la canción y/o beatmap de una manera llamativa. También pueden servir como título o encabezamiento para agregar estilo y calidad. Para saber cómo agregar una imagen, consulta la [sección de imágenes del artículo de BBCode](/wiki/BBCode#imágenes).
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 7]
```diff
@@ -1,25 +1,25 @@
...
 
 ### Secciones de dificultad
 
-Una *sección de dificultad* es una lista de todas las dificultades de invitados (a menudo junto con los íconos respectivos) y sus creadores ubicados en una descripción del beatmap. Se utiliza sobre todo para dar crédito a los mapeadores invitados.
+Una *sección de dificultad* es una lista de todas las dificultades de invitados (a menudo, junto con los íconos respectivos) y sus creadores ubicados en una descripción del mapa. Se utiliza sobre todo para dar crédito a los mapeadores invitados.
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:23]:
```suggestion
Una *sección de dificultad* es una lista de todas las dificultades de invitados (a menudo junto con los íconos respectivos) y sus creadores ubicados en una descripción del beatmap. Se utiliza sobre todo para dar crédito a los mapeadores invitados.
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 8]
```diff
@@ -1,25 +1,25 @@
 ---
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:1]:
```suggestion
```

File: wiki/Beatmap/Beatmap_description/es.md [hunk 9]
```diff
@@ -1,25 +1,25 @@
...
-no_native_review: true
+no_native_review: false
 ---
```
Comment (agatemosu) [inline wiki/Beatmap/Beatmap_description/es.md:4]:
```suggestion

```

File: wiki/Beatmap/Beatmap_description/es.md (hunk 6 above)
Comment (agatemosu) [reply wiki/Beatmap/Beatmap_description/es.md:19]: no creo que se deban usar las comas aquí

File: wiki/Beatmap/Beatmap_description/es.md (hunk 7 above)
Comment (agatemosu) [reply wiki/Beatmap/Beatmap_description/es.md:23]: no creo que se deban usar las comas aquí

**Review by TPGPL — APPROVED** [review]

**Pull request merged**